input/sensors/src/sensors.c

```diff
@@ -122,21 +165,84 @@ _j4status_sensors_feature_free(gpointer data)
     g_free(feature);
 }
 
-static void
-_j4status_sensors_add_feature_temp(J4statusPluginContext *context, const sensors_chip_name *chip, const sensors_feature *feature)
+static char *
+_j4status_sensors_get_feature_name(const sensors_chip_name *chip, const sensors_feature *feature)
 {
     int n;
-    char name[MAX_CHIP_NAME_SIZE + strlen(feature->name) + 1];
+    char *name = malloc(MAX_CHIP_NAME_SIZE + strlen(feature->name) + 1);
```

Comment:
I do not want allocation here (and that would be `g_<something>`, not `malloc`).
`feature->name` is the file part of a sysfs path, so just use `static char name[MAX_CHIP_NAME_SIZE + NAME_MAX + 1];`. We do not thread, so it is safe to use a static variable here.